docs/source/guide/s3.rst

```diff
@@ -497,10 +497,7 @@ To generate a pre-signed URL, use the
     # to send the GET, but we will use requests here to keep things simple.
     response = requests.get(url)
 
-If your bucket requires the use of signature version 4, you can elect to use it
-to sign your URL. This does not fundamentally change how you use generator,
-you only need to make sure that the client used has signature version 4
-configured.::
+The boto client will generate a sigv4 (s3v4) signature by default, but it may be changed by doing::
```

Comment:
Let's clarify this behavior.  We default to sigv4 for new regions that are sigv4 only, but we default to the older signature version for backwards compatibility.  We can use the endpoint data to categorize these regions: https://github.com/boto/botocore/blob/develop/botocore/data/endpoints.json#L1965